### ISSUE TITLE
@remix-run/cloudflare session storage creation functions to not accept `Data` / `FlashData` type arguments

### ISSUE THREAD

**Issue by johtso** [remix-run/remix#5636]
### What version of Remix are you using?

1.14.0

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

I'm using `createCookieSessionStorage` and `createWorkersKVSessionStorage` from `@remix-run/cloudflare`, but they don't seem to accept any type arguments so I can't describe the shape of my session data and my flash data.

~~[The example in the docs](
https://github.com/remix-run/remix/blob/9a9c955f241c5742d227542e13c7387b99d4628b/docs/utils/sessions.md?plain=1#L37) does `createCookieSessionStorage<SessionData, SessionFlashData>(` but those type arguments don't seem to exist.~~

https://github.com/remix-run/remix/blob/9a9c955f241c5742d227542e13c7387b99d4628b/docs/utils/sessions.md?plain=1#L37

~~How am I supposed to describe the shape of my persisted session data and my flashed session data?
Did this type arguments exist at one point?~~

~~I'm guessing that should just be removed from the example for now?~~

### Expected Behavior

N/A

### Actual Behavior

N/A

**Comment by dmarkow** [comment]
The type definitions are right above the line you linked to:

```ts
type SessionData = {
  userId: string;
};
type SessionFlashData = {
  error: string;
};
```

So you can customize those types and name them whatever you want, or even just inline them:

```ts
type SessionData = { custom: string };
type SessionFlashData = { other: string };
const { getSession } = createCookieSessionStorage<SessionData, SessionFlashData>({ cookie: ... });

// Or just inline
const { getSession } = createCookieSessionStorage<{custom: string}, {other: string}>({ cookie: ... });

// Then when you use the session later, it will have these types:
session.get("string"); // Type is string
```

**Comment by johtso** [comment]
@dmarkow aha, I just realised where the confusing is coming from. I'm importing `createCookieSessionStorage` from `@remix-run/cloudflare` and that version does not accept any type arguments, so it's throwing a type error. The same would go for `createWorkersKVSessionStorage`.

**Comment by johtso** [comment]
Oops! Turns out I was using `1.13.0`.. and this was fixed in `1.14.0`. Classic! https://github.com/remix-run/remix/pull/1830
